samples/snippets/pom.xml

```diff
@@ -113,6 +113,12 @@
       <version>1.1.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-core</artifactId>
```

Comment:
We should not depend on beam sdks here, since this would create a dependency cycle.

Reply:
Good point, didn't realize that